source/reference/minio-mc-admin/mc-admin-update.rst

```diff
@@ -88,3 +88,13 @@ Syntax
    :optional:
 
    Pass this flag to confirm the update and bypass the confirmation prompt.
+
+Behavior
+--------
+
+Binary Compression 
+~~~~~~~~~~~~~~~~~~
+
+Starting with :minio-release:`RELEASE.2024-01-28T22-35-53Z`, :mc-cmd:`mc admin update` compresses the binary before sending to all nodes in the deployment.
```

Comment:
Why not use the 
```
.. versionchanged:: ####
```

Syntax?

Reply:
```suggestion
.. versionchanged:: RELEASE.2024-01-28T22-35-53Z

   :mc-cmd:`mc admin update` compresses the binary before sending to all nodes in the deployment.
```

Reply:
If you do it here, you could also apply the same syntax to the other files where you have the `Starting with` paragraphs.